orchagent/pfcwdorch.cpp

```diff
@@ -818,27 +879,89 @@ void PfcWdSwOrch<DropHandler, ForwardHandler>::doTask(swss::NotificationConsumer
 
             entry->second.handler->commitCounters();
             entry->second.handler = nullptr;
+            // Remove storm status in APPL_DB for warm-reboot purpose
+            string key = APP_PFC_WD_TABLE_NAME ":" + entry->second.portAlias;
+            m_applDbRedisClient.hdel(key, to_string(entry->second.index));
         }
     }
     else
     {
         SWSS_LOG_ERROR("Received unknown event from plugin, %s", event.c_str());
+        return false;
     }
+
+    return true;
 }
 
 template <typename DropHandler, typename ForwardHandler>
-void PfcWdSwOrch<DropHandler, ForwardHandler>::doTask(SelectableTimer &timer)
+bool PfcWdSwOrch<DropHandler, ForwardHandler>::bake()
 {
-    SWSS_LOG_ENTER();
+    // clean all *_last fields in COUNTERS_TABLE
```

Comment:
>// clean all *_last fields in COUNTERS_TABLE [](start = 4, length = 44)

Is this a new design? Do you have document?

Reply:
No, I do not have document. I see counter values are still in the COUNTERS_DB after warm-reboot. This makes sure that when PFCWD starts the detect and restore state machines it starts with the same state machine state as cold reboot.